docs/code-quality/ca2248.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "CA2248: Provide correct enum argument to Enum.HasFlag"
+ms.date: 05/06/2020
+ms.topic: reference
+f1_keywords:
+- CA2248
+helpviewer_keywords:
+- CA2248
+author: mavasani
```

Comment:
@mavasani Manish, we need an internal name for author. Is it best to use yours? If it's important to know whether you were the original contributor or not, we could use my name instead.

Reply:
Anything works for me. However, note that this PR should be considered (and probably tagged in the title too) as a Work in progress [WIP]. @Evangelink went an extra mile to follow our new documentation guidelines for new CA rules and created a doc PR for this new rule, while the analyzer PR for this is in review: https://github.com/dotnet/roslyn-analyzers/pull/3557#issuecomment-624914045

Lets wait for that PR to be merged before reviewing/merging this.

Reply:
Yes! Thanks, @Evangelink